# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/modules/RemoteWebProgress.jsm
# Commit: e1fbefe38944
# Full Hash: e1fbefe38944eb82d3c2e87a0307b121daf0e830
# Author: Barret Rennie <barret@brennie.ca>
# Date: 2019-04-04 04:33:36
# Regressor Bug: 1510569
# File Overlap Count: 1
# Description:
#   Bug 1510569 - Port onProgressChange notifications inside WebProgressChild.jsm to C++ r=baku
#   
#   We do not need to handle onProgressChange64 notifications since the TabChild's
#   web progress events are filtered through an nsBrowserStatusFilter, which
#   truncates onProgresChange64 event values to 32-bit integers and then calls
# ==============================================================================

diff -r 102881458323 -r e1fbefe38944 toolkit/modules/RemoteWebProgress.jsm
--- a/toolkit/modules/RemoteWebProgress.jsm	Wed Apr 03 17:32:21 2019 +0000
+++ b/toolkit/modules/RemoteWebProgress.jsm	Wed Apr 03 17:32:41 2019 +0000
@@ -31,7 +31,6 @@
       this._messageManager.removeMessageListener("Content:StateChange", this);
       this._messageManager.removeMessageListener("Content:LocationChange", this);
       this._messageManager.removeMessageListener("Content:SecurityChange", this);
-      this._messageManager.removeMessageListener("Content:ProgressChange", this);
       this._messageManager.removeMessageListener("Content:LoadURIResult", this);
     }
 
@@ -40,7 +39,6 @@
     this._messageManager.addMessageListener("Content:StateChange", this);
     this._messageManager.addMessageListener("Content:LocationChange", this);
     this._messageManager.addMessageListener("Content:SecurityChange", this);
-    this._messageManager.addMessageListener("Content:ProgressChange", this);
     this._messageManager.addMessageListener("Content:LoadURIResult", this);
   }
 
@@ -249,13 +247,6 @@
 
       this.onSecurityChange(webProgress, request, state);
       break;
-
-    case "Content:ProgressChange":
-      this.onProgressChange(
-        webProgress, request, json.curSelf, json.maxSelf, json.curTotal,
-        json.maxTotal
-      );
-      break;
     }
   }
 }